packages/realm/src/Realm.ts

```diff
@@ -194,7 +194,7 @@ type RealmSchemaExtra = Record<string, ObjectSchemaExtra | undefined>;
 type ObjectSchemaExtra = {
   constructor?: RealmObjectConstructor;
   defaults: Record<string, unknown>;
-  // objectTypes: Record<string, unknown>;
+  // ObjectTypes: Record<string, unknown>;
```

Comment:
Admittedly, this line should probably be deleted, but I can see how the `capitalized-comments` rule would be annoying when commenting out code while developing and having format-on-save mess it up like this.